### PR TITLE
Update offering-PCI-DSS.md

### PR DESCRIPTION
**Pull request by tymconn**
Adding Microsoft Defender Advanced Threat Protection to the list of PCI Certified products as it achieved certification in the latest Azure PCI Audit.

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/compliance/offering-PCI-DSS.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ It is important to understand that PCI DSS compliance status for Azure, OneDrive
 - Flow cloud service either as a standalone service or as included in an Office 365 or Dynamics 365 branded plan or suite
 - Graph
 - Intune
+- [Microsoft Defender Advanced Threat Protection](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/microsoft-defender-advanced-threat-protection)
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/offering-PCI-DSS.md]:
```suggestion
- [Microsoft Defender Advanced Threat Protection](https://docs.microsoft.com/windows/security/threat-protection/microsoft-defender-atp/microsoft-defender-advanced-threat-protection)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - please remove locale from link.

**Comment by tymconn** [comment]
Thank you for the tip Johan, I will do that in the future as well.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by mypil** [comment]
@robmazz  - PR has been copy edited and is ready for final review. 

Thank you.

cc: @kenwith

**Comment by robmazz** [comment]
Confirmed change and audit date with CELA - will merge PR now.

**Pull request merged**